src/snowflake/snowpark/functions.py

```diff
@@ -8536,3 +8536,15 @@ def locate(expr1: str, expr2: ColumnOrName, start_pos: int = 1) -> Column:
     _substr = lit(expr1)
     _str = _to_col_if_str(expr2, "locate")
     return builtin("charindex")(_substr, _str, lit(start_pos))
+
+
+def enable_open_telemetry():
```

Comment:
small nit: should this say `enable_tracing()`? opentelemetry is also logs and metrics, I think the intention is only to control tracing, correct?

Reply:
What about `enable_span_record()`, I think this is more specific about what the function do

Reply:
Yeah makes sense, as long as it's scoped to tracing concepts.

Reply:
cc @sfc-gh-bli Should we have similar API on Java side as well?